### ISSUE TITLE
underlying database names

### ISSUE THREAD

**Issue by petersamokhin** [edgedb/edgedb#5592]
hello there! tl;dr is there a way to control the underlying database names?
currently we see:
```
 V2f147ded60___edgedbtpl__
 V2f147ded60___edgedbsys__
 V2f147ded60_edgedb
```

This is not possible to use with the managed DB instances where the databases are pre-created.
However, we could pre-create multiple databases once and provide the EdgeDB instance with their names.
Is that supported? If not, can it be supported in the future?

**Comment by elprans** [comment]
EdgeDB supports a "single database" mode, whereby all metadata is stored in a single database and the `CREATE DATABASE` statement is disabled, so you only get the `edgedb` database.  This mode is normally auto-detected on startup.  Are you seeing a different behavior?

**Comment by petersamokhin** [comment]
@elprans could you please document this behaviour somewhere? 
How is it determined exactly? e.g. by trying to create a database and falling back to the single database named `edgedb`?
Is it possible to control the default/fallback db name?

**Comment by elprans** [comment]
> by trying to create a database and falling back to the single database

Yes

>  Is it possible to control the default/fallback db name?

In single db mode EdgeDB uses the database provided in `--backend-dsn` (`EDGEDB_SERVER_BACKEND_DSN`), e.g. if you give it `postgres://host:port/mydb` it'll use `mydb`.

**Comment by petersamokhin** [comment]
@elprans  Thank you for clarification! The last question: if we have the postgres backups, that is used as the backend for the edgedb instance, do we still need to backup (dump) edgedb? Does edgedb store anything anywhere else apart from the postgres?

**Comment by petersamokhin** [comment]
Perfect! Thanks a lot.

**Comment by petersamokhin** [comment]
hey @elprans, just realised that the last time I got it wrong.

> and the CREATE DATABASE statement is disabled, so you only get the edgedb database. 

I thought that there will be only a single underlying postgres database, but >1 databases in edgedb.
i.e. it is not really clear that each edgedb database requires a separate&dedicated postgres database.

could you please confirm?

**Comment by elprans** [comment]
> each edgedb database requires a separate&dedicated postgres database.

Yes.  If you don't have a permission to `CREATE DATABASE` in Postgres, `CREATE DATABASE` in EdgeDB will likewise not be available.

**Comment by elprans** [comment]
To clarify, the underlying database name in single-database mode would be whatever you specified in `--backend-dsn`
